backend/src/main/scala/cromwell/backend/wdl/FileSizeLimitationConfig.scala

```diff
@@ -0,0 +1,48 @@
+package cromwell.backend.wdl
+
+import com.typesafe.config.ConfigFactory
+import net.ceedubs.ficus.readers.ValueReader
+import net.ceedubs.ficus.Ficus._
+
+trait FileSizeLimitationConfig {
+
+  def readLinesLimit: Int
+
+  def readBoolLimit: Int
+
+  def readIntLimit: Int
+
+  def readFloatLimit: Int
+
+  def readStringLimit: Int
+
+  def readJsonLimit: Int
+
+  def readTsvLimit: Int
+
+  def readMapLimit: Int
+
+  def readObjectLimit: Int
+}
```

Comment:
it could and perhaps should, but because we're using traits and mixins the trait is more desirable.

If it were a case class it would have to be the _only_ case class being mixed in at the top, and that's not really a necessary requirement for this to work.